handlers/assign_reviewer/tests/no-reviewer.json

```diff
@@ -0,0 +1,207 @@
+{
+  "initial": {
+  },

```

Comment:
Let's set an initial assignee and ensure that they're still set in the expected values.